src/record.ts

```diff
@@ -27,12 +28,14 @@ function isPureTypemap(typemap: object) {
   return true
 }
 
-function internalRecord(
-  typemap: { [key: string]: Runtype<any> | OptionalRuntype<any> },
-  sloppy: boolean,
+export function internalRecord(
```

Comment:
I think the changes here are pretty straightforward: basically replacing sloppy with nonStrict, and adding`isNonStrict = isNonStrictRuntypeSymbol` so that we can 'pass on' the strictness when using things like `omit`, etc.